### PR TITLE
fix text size and update hero image

### PR DESCRIPTION
**Pull request by norachams**

### CODE REVIEW

File: src/components/sections/Hero.section.tsx [hunk 1]
```diff
@@ -34,18 +34,16 @@ const HeroSection: React.FC = () => {
...
-            <p className="pb-3 text-sm font-thin [word-spacing:35px] mr-[-0.5rem]">Days Hrs Mins Secs</p>
-            <button className="bg-mint text-white px-4 py-1 rounded">
+            <p className="mx-5 space-x-8 text-base font-bold [word-spacing:45px]">{countdown}</p>
+            <p className="pb-3 text-sm mb-2 font-thin [word-spacing:35px] mr-[-0.5rem]">Days Hrs Mins Secs</p>
+            <button className="bg-mint text-white mb-3 px-4 py-2 rounded font-bold [letter-spacing:2px]">
               REGISTER
             </button>
-            <img  className="mt-[-4rem]" src={HeroIllustration} alt="Hero Illustration" />
-            <div className="flex items-center justify-center space-x-4 mt-[-8.5rem]">
+            <img  className="mt-[-0.5rem]" src={HeroIllustration} alt="Hero Illustration" />
```
Comment (juancwu) [inline src/components/sections/Hero.section.tsx:45]:
```suggestion
            <img  className="-mt-2" src={HeroIllustration} alt="Hero Illustration" />
```

Comment (juancwu) [reply src/components/sections/Hero.section.tsx:45]: Tailwind has classes for negative values. Little tip here: all measures in tailwind start with 0.25rem. So mt-1 = 0.25rem and mt-2 = 0.5rem.

Comment (juancwu) [reply src/components/sections/Hero.section.tsx:45]: You can also use half points such as `mt-0.5` to get 0.125rem

**Review by juancwu — CHANGES_REQUESTED** [review]
Could you match the sizes with the ones defined in `src/index.css`? Some text became hard to read because now it is too small, such as the place, time area in the hero section.

You can use use arbitrary values such as `text-[1.875rem]` for now because I can't think of a good name for classes for the font sizes right now. Ideally, with a set of classes by extending the tailwind config would be the ideal.

**Review by juancwu — APPROVED** [review]

File: src/components/sections/Hero.section.tsx [hunk 2]
```diff
@@ -34,18 +34,16 @@ const HeroSection: React.FC = () => {
...
-            <p className="pb-2 text-lg">Wilfrid Laurier’s first large-scale global hackathon!</p>
-            <p className="pb-3 text-xs font-thin [word-spacing:5px]">Waterloo, ON | April 6th - 8th | In-Person</p>
-            <p className="mx-5 space-x-8 text-2sm font-bold [word-spacing:45px]">{countdown}</p>
-            <p className="pb-3 text-sm font-thin [word-spacing:35px] mr-[-0.5rem]">Days Hrs Mins Secs</p>
-            <button className="bg-mint text-white px-4 py-1 rounded">
+            <h1 className="font-bold text-mint">&lt;HawkHacks 2024/&gt;</h1>           
+            <p className="pb-2 text-xl">Wilfrid Laurier’s first large-scale global hackathon!</p>
+            <p className="pb-3 text-sm font-thin [word-spacing:5px]">Waterloo, ON | April 6th - 8th | In-Person</p>
+            <p className="mx-5 space-x-8 text-base font-bold [word-spacing:45px]">{countdown}</p>
+            <p className="pb-3 text-sm mb-2 font-thin [word-spacing:35px] mr-[-0.5rem]">Days Hrs Mins Secs</p>
```
Comment (teoh4770) [inline src/components/sections/Hero.section.tsx:41]: change mr-[-0.5rem] to -mr-2

Comment (teoh4770) [reply src/components/sections/Hero.section.tsx:41]: sorry for being picky @norachams

File: src/components/sections/Hero.section.tsx [hunk 3]
```diff
@@ -34,11 +34,11 @@ const HeroSection: React.FC = () => {
     return (
         <div className="bg-midnight text-white py-16">
           <div className="text-center">
-            <h1 className="font-bold text-mint">&lt;HawkHacks 2024/&gt;</h1>           
-            <p className="pb-2 text-xl">Wilfrid Laurier’s first large-scale global hackathon!</p>
-            <p className="pb-3 text-sm font-thin [word-spacing:5px]">Waterloo, ON | April 6th - 8th | In-Person</p>
-            <p className="mx-5 space-x-8 text-base font-bold [word-spacing:45px]">{countdown}</p>
-            <p className="pb-3 text-sm mb-2 font-thin [word-spacing:35px] mr-[-0.5rem]">Days Hrs Mins Secs</p>
+            <h1 className="font-bold text-mint text-7xl">&lt;HawkHacks 2024/&gt;</h1>
```
Comment (teoh4770) [inline src/components/sections/Hero.section.tsx:37]: good fix overall! just try to add a bit spacing at the bottom, seems a bit tight here

Comment (teoh4770) [reply src/components/sections/Hero.section.tsx:37]: and then you are good

**Review by teoh4770 — APPROVED** [review]
good job!

**Pull request merged**